bootstrap/kubeadm/main.go

```diff
@@ -144,6 +146,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
 
+	fs.Float32Var(&clusterCacheTrackerClientQPS, "clustercachetracker-client-qps", 20,
+		"Maximum queries per second from the controller client to the Kubernetes API server of workload clusters. Defaults to 20")
```

Comment:
the `Defaults to x` part is not needed for these new and existing entries, FWIW
they will be printed on the CLI --help based on the `, 20,`.

Reply:
ack, how it currently looks like:

```sh
$ go run ./bootstrap/kubeadm/ -h
...
      --kube-api-burst int                     Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30 (default 30)
      --kube-api-qps float32                   Maximum queries per second from the controller client to the Kubernetes API server. Defaults to 20 (default 20)
...
```

Reply:
Yup, you're right good catch. Works as long as the default is not the zero value (it's tricky with bool :D)

Reply:
Fixed